.github/workflows/enso4igv.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Enso Language Support for IGV
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        java: [ '8' ]
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Java
+      uses: actions/setup-java@v2
+      with:
+        distribution: 'zulu'
+        java-version: ${{ matrix.java }}        
+        cache: maven
+    - name: Build with Maven
+      run: mvn -B package --file tools/enso4igv/pom.xml
```

Comment:
If we're putting the plugin in the enso repository I'd like it to be a part of the main SBT build configuration. Ideally. I understand that it's a bigger task, and we can implement the integration later.

Reply:
I understand that introducing another build system into the repository is increasing entropy, but we cannot proceed without Maven. 

The problem is that the build heavily relies on `org.apache.netbeans.utilities:nbm-maven-plugin:4.7` maven plugin to package the `NBM` file properly. I don't know any other way to invoke such functionality from `sbt` other than to execute Maven `mvn` executable. Having `sbt` command to invoke `mvn` seems kind a use less...

Reply:
I would agree that keeping everything in SBT would be best, but duplicating https://github.com/apache/netbeans-mavenutils-nbm-maven-plugin/ is also not an option :(